less/overrides.less

```diff
@@ -15,6 +15,10 @@
     outline: none !important;
 
     .mq-editable-field {
+        .mq-root-block {
+            overflow-x: scroll !important;
```

Comment:
It's been a while since I've done much CSS- do we have to use the !important override here or is there somewhere else we can override the style without having to force priority?

Reply:
Looks like we actually don't need the `!important` part here. (I added it out of habit because Aphrodite does that too.) Thanks for pointing that out!